src/engine/controls/cuecontrol.cpp

```diff
@@ -363,6 +363,8 @@ void CueControl::trackLoaded(TrackPointer pNewTrack) {
     }
     m_pLoadedTrack = pNewTrack;
 
+    connect(m_pLoadedTrack.get(), &Track::analyzed, this, &CueControl::trackAnalyzed, Qt::DirectConnection);
```

Comment:
Sorry, but clang-format won't let me keep it. But if you like, I can change the others as well for consistency.

Reply:
Ah, OK. In that case, never mind.

Reply:
`clang-format` allows two wrapping styles: Either wrap after each argument or never wrap. I suggest the former.

Reply:
I'd prefer to table clang-format discussions for post-2.3.  We have an awkward configuration that is not well supported right now and produces some weird results like this.  Rather than bogging down every PR with style discussions, let's keep formatting questions separate for now.

Reply:
Ok, so I will leave this as it is now.